src/model.cpp

```diff
@@ -527,6 +527,44 @@ bool Model::fixVariableInterfaces()
     return allOk;
 }
 
+static const size_t MAX_SIZE_T = std::numeric_limits<size_t>::max();
```

Comment:
Might want to move this to `utilities.h` and also remove the one that is defined in `analyser.cpp`?

Reply:
fcb43fc